### DISCUSSION TITLE
gameSchema

### DISCUSSION THREAD

**Discussion by Sensinki** [Sensinki/Blok-Tech-Team-4A#15] [Ideas]
Ik denk dat we voor gameSchema onderstaande model kan gebruiken voor ons database. Wat denken jullie hiermee? eheh :))

const userSchema = new mongoose.Schema({
  username: String,
  password: String,
  email: String,
  likedGames: [
    {
      "name" : "Minecraft",
      "image" : "url"
    }, 
    {
      "name" : "Valorant",
      "image" : "url"
    }, 
  ]
});

**Comment by D0nnaz** [comment]
Ja, misschien handig om hier ook al een Boolean met liked toe te voegen (kan gebruikt worden om de juiste chats in beeld te laten komen). Misschien ook iets met matched en dan een Boolean.

match = false 

 

dat geef je door aan de db

 

als de user het doorlopen heeft match = true

 

if match = false

res.redirect /match

 

if match = true

red.redirect /

**Comment by D0nnaz** [comment]
Deze discussion kunnen we denk ik wel sluiten toch @Sensinki?

**Reply by Sensinki** [reply]
oh ja tuurlijk
